Sources/ArcGISToolkit/Components/Offline/OnDemand/OnDemandMapAreaSelectorView.swift

```diff
@@ -228,8 +223,11 @@ struct OnDemandMapAreaSelectorView: View {
     
     /// The handle view for the map area selector.
     struct Handle: View {
+        let location: HandleLocation
         /// The position of the handle.
         let position: CGPoint
+        /// The corner radius of the handle.
+        let cornerRadius: CGFloat
         /// The closure to call when the map area selector should be resized.
         let resize: (CGPoint) -> Void
```

Comment:
You can pass the handle location enum here as well, so that we don't have to hard-code it in the closure where we are showing the handle.